docs-src/changelog.md

```diff
@@ -2,6 +2,10 @@
 HypoFuzz uses [calendar-based versioning](https://calver.org/), with a
 `YY-MM-patch` format.
 
+## 25.04.9
+
+Improve uniqueness of database keys for tests which use `@pytest.mark.parametrize`.
```

Comment:
It's not just uniqueness; it's ensuring that Hypofuzz uses the same db keys as Hypothesis, so that replay works!